### PR TITLE
Add a first version of an Intro to Bowtie post / announcement.

### PR DESCRIPTION
**Pull request by Julian**
This is a first draft, I have to read through for gaps in content and awkwardness, but getting it up ASAP for comments, since it's already taken me too long to get it "done".

So yes, comments welcome please from anyone either on what's here or what's not here!

### CODE REVIEW

**Comment by gregsdennis** [comment]
I feel like there needs to be some mention of the [JSON Path Comparison project](https://cburgmer.github.io/json-path-comparison/).  You had created an [issue](https://github.com/json-schema-org/JSON-Schema-Test-Suite/issues/314) for this idea initially, but (a few months) later I shared a link to the JSON Path site as a working example in another space.  I don't know how much inspiration you drew from that, but to me, it seems like a catalyst.

**Comment by gregsdennis** [comment]
Title and header seem to not be rendered right
image

File: pages/posts/bowtie-intro.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+---
+title: ""
```
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:2]: "Introducing: Bowtie!" perhaps?

File: pages/posts/bowtie-intro.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  - News
+  - validation
+type: Engineering
+cover: /img/posts/2022/bowtie-intro/cover.webp
+authors:
+  - name: Julian Berman
+    photo: /img/avatars/julian.webp
+    link: https://twitter.com/JulianWasTaken
+    byline: JSON Schema Test Suite Maintainer & Technical Lead, API Specifications @Postman
+excerpt: "Bowtie: a new tool for executing JSON Schema implementations & a call to help improve it"
```
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:15]: then remove "Bowtie:" from here

**Review by jdesrosiers — APPROVED** [review]

File: pages/posts/bowtie-intro.md [hunk 3]
```diff
@@ -12,24 +12,44 @@ authors:
...
+Does every *implementation* of the specification, across the wide spectrum of programming languages, properly *follow* the specification?
+What about for other more complicated schema / instance pairs?
+This is the key question Bowtie tries to address -- how can we compare behavioral differences between JSON Schema implementations and between the specification, in the hopes that we make it easier to fix implementation issues and clarify any unclear portions of the specification?
+
 ## What It Is
 
 [Bowtie](https://github.com/bowtie-json-schema/bowtie/) is what I'm calling a "meta-validator", by which I mean a program which can execute *other* JSON Schema validation implementations and collect results from them.
 
+There's prior art in doing this sort of thing.
+The [JSON Path Comparison project](https://cburgmer.github.io/json-path-comparison/) does this extremely well for [JSONPath](https://goessner.net/articles/JsonPath/), and [Nicolas Seriot's "Parsing JSON is a Minefield"](https://seriot.ch/projects/parsing_json.html) is a fantastic example for JSON itself.
```
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:43]: I'm not hung up on it, but the official spec is being developed at https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base.  Alternatively this link can be used.  Although, the Goessner post is more well-known.  Just an idea.  I leave it to your preference.

Comment (Julian) [reply pages/posts/bowtie-intro.md:43]: I have no preference, I took the link from the comparison page (which links to that Goessner one) -- happy to update it, I don't know anything about JSONPath's development but certainly trust you :D

Comment (Julian) [reply pages/posts/bowtie-intro.md]: (Done)

**Review by gregsdennis — APPROVED** [review]

**Comment by Julian** [comment]
Ben's happy to read this after it's live :D, hitting merge (even though the date is technically yesterday, but yeah whatever, lies and the internet, what better pairing!).

**Pull request merged**